native/src/main/java/io/ballerina/stdlib/sql/utils/Utils.java

```diff
@@ -1178,4 +1178,20 @@ public static String getBTypeName(Type ballerinaType) {
         }
         return ballerinaType.getName();
     }
+
+    public static boolean isKnownRecordType(Type ballerinaType) {
+        String[] knownRecordTypes = {
```

Comment:
Let's make this a util class constant. Do we only support civil and utc? I thought TimeOfDay and others are also supported.

Reply:
Made a class constant in 2bf215b

Civil, Date and TimeOfDay are supported